tests/lib/rules/comma-spacing.js

```diff
@@ -99,13 +99,13 @@ eslintTester.addRuleTest("lib/rules/comma-spacing", {
         {code: "var arr = [1,];", args: [2, {before: false, after: false}]},
         {code: "var arr = [,2];", args: [2, {before: false, after: false}]},
         {code: "var arr = [1,2];", args: [2, {before: false, after: false}]},
-        {code: "var arr = [,,];", args: [2, {before: false, after: false}]},
-        {code: "var arr = [1,,];", args: [2, {before: false, after: false}]},
+        {code: "var arr = [,];", args: [2, {before: false, after: false}]},
+        {code: "var arr = [1,];", args: [2, {before: false, after: false}]},
         {code: "var arr = [,2,];", args: [2, {before: false, after: false}]},
-        {code: "var arr = [,,3];", args: [2, {before: false, after: false}]},
+        {code: "var arr = [,3];", args: [2, {before: false, after: false}]},
         {code: "var arr = [1,2,];", args: [2, {before: false, after: false}]},
         {code: "var arr = [,2,3];", args: [2, {before: false, after: false}]},
-        {code: "var arr = [1,,3];", args: [2, {before: false, after: false}]},
+        {code: "var arr = [1,3];", args: [2, {before: false, after: false}]},
         {code: "var arr = [1,2,3];", args: [2, {before: false, after: false}]},
```

Comment:
There now seem to be duplicated tests here:

- 99/103
- 100/105
- 101/108

Was this intended?